doc/next-steps/filter-examples.md/$eq.md

```diff
@@ -0,0 +1,94 @@
+# $eq
+
+Records that exactly match the given value will be returned.
```

Comment:
should we mention that this is case sensitive? `workout` will not match `Workout`, right?

Reply:
true